cat-dog-app/src/App.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import NavBar from './components/NavBar';
+import Dog from './components/Dog';
+import Cat from './components/Cat';
+import RandomDogByBreed from './components/MultipleRandomDogs';
+import MultipleRandomDogs from './components/MultipleRandomDogs';
+import MultipleRandomCat from './components/MultipleRandomCats';
+
+import { Route } from 'react-router-dom'
+
+
+import './App.css';
+
+function App() {
+  return (
+    <div className="App">
+      <NavBar/>
```

Comment:
Importing the `<Switch>` Component from `react-router-dom` and then wrapping all your routes with it allows you to be able to error handle if there are no component that deal with that route. This would be the equivalent of adding a 404 page.